hbase-examples/pom.xml

```diff
@@ -159,6 +159,8 @@
     <dependency>
       <groupId>com.github.stephenc.findbugs</groupId>
```

Comment:
Don't need to remove this stephenc findbugs? You do above in hbase-client/pom.xml  You seem to leave it in elsewhere so guessing it ok.

Reply:
For hbase-client and another module, the problem is that we declare this dependency only under the hadoop 2 profile and it will cause compile error with hadoop 3. So just move it to the general dependencies section.